### PR TITLE
<DO NOT MERGE - In Progress> add support for oryx build end point

### PR DESCRIPTION
**Pull request by arroyc**
- [X] The purpose of this PR is explained in this message or in an issue. 
- [ ] Tests are included and/or updated for code changes. To do
- [X] Proper license headers are included in each file.

### CODE REVIEW

File: src/BuildServer/BuildServerResponse.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+// --------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    enum BuildState
+    {
+        Success,
+        Running,
```
Comment (william-msft) [inline src/BuildServer/BuildServerResponse.cs:13]: Nit: Maybe we should consider to rename to `Building`. Running sounds like it like the built app is running to me.

File: src/BuildServer/BuildServerRequests.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    public class BuildServerRequests
+    {
```
Comment (william-msft) [inline src/BuildServer/BuildServerRequests.cs:11]: Would be nice to have a documentation link to a JSON sample

File: src/BuildServer/BuildServerResponse.cs (hunk 1 above)
Comment (arroyc) [reply src/BuildServer/BuildServerResponse.cs:13]: I like Building .. good one

File: src/BuildServer/BuildServerRequests.cs (hunk 2 above)
Comment (arroyc) [reply src/BuildServer/BuildServerRequests.cs:11]: of course we should .. good point

**Pull request closed without merging**